File: src/com/l2kt/gameserver/network/serverpackets/PrivateStoreMsgSell.java
package com.l2kt.gameserver.network.serverpackets;

import com.l2kt.gameserver.model.actor.instance.Player;

public class PrivateStoreMsgSell extends L2GameServerPacket
{
	private final Player _activeChar;
	private String _storeMsg;
	
	public PrivateStoreMsgSell(Player player)
	{
		_activeChar = player;
		if (_activeChar.getSellList() != null)
			_storeMsg = _activeChar.getSellList().getTitle();
	}
	
	@Override
	protected final void writeImpl()
	{
		writeC(0x9c);
		writeD(_activeChar.getObjectId());
		writeS(_storeMsg);
	}
}

File: src/com/l2kt/gameserver/network/serverpackets/PrivateStoreMsgSell.kt
package com.l2kt.gameserver.network.serverpackets

import com.l2kt.gameserver.model.actor.instance.Player

class PrivateStoreMsgSell(private val _activeChar: Player) : L2GameServerPacket() {
    private var _storeMsg: String? = _activeChar.sellList?.title

    override fun writeImpl() {
        writeC(0x9c)
        writeD(_activeChar.objectId)
        writeS(_storeMsg)
    }
}